tests/test_base.py

```diff
@@ -364,15 +365,67 @@ def test_getfloat_defaults(self):
         getter = getconf.ConfigGetter('TESTNS')
         self.assertEqual(0.0, getter.getfloat('test'))
         self.assertEqual(42.3, getter.getfloat('foo', 42.3))
-        self.assertEqual(None, getter.getfloat('test', None))
+        self.assertIsNone(getter.getfloat('test', None))
 
     def test_getfloat_defaults_raises(self):
         """Test fetching from the default config dict with non-float values."""
         getter = getconf.ConfigGetter('TESTNS')
-        self.assertRaises(AssertionError, lambda: getter.getfloat('test', False))
-        self.assertRaises(AssertionError, lambda: getter.getfloat('test', 42))
-        self.assertRaises(AssertionError, lambda: getter.getfloat('test', '4.2'))
-        self.assertRaises(AssertionError, lambda: getter.getfloat('test', (1, )))
+        self.assertRaises(AssertionError, getter.getfloat, 'test', False)
+        self.assertRaises(AssertionError, getter.getfloat, 'test', 42)
+        self.assertRaises(AssertionError, getter.getfloat, 'test', '4.2')
+        self.assertRaises(AssertionError, getter.getfloat, 'test', (1, ))
```

Comment:
👍 on removing all those lambdas, but the part that's not relevant to the current PR should be done in a separate commit